packages/playground/src/App.vue

```diff
@@ -253,6 +255,27 @@ onMounted(async () => {
   openProfile.value = true;
 });
 
+/**
+ * Initializes the routes with update information.
+ *
+ * This function is called when the component is mounted.
+ * It iterates over the routes and their items, checking if the item has an update.
+ * If the item has an update, it calculates the next 30 days from the item's timestamp
+ * and sets the hasUpdate property to true if the current time is less than the calculated timestamp.
+ */
+
+onMounted(() => {
+  const next30DaysInMs = 30 * 24 * 60 * 60 * 1000;
+  routes.map(route => {
+    route.items.map(item => {
+      const next30DaysFromNow = item.timestamp ? item.timestamp + next30DaysInMs : 0;
+      if (item.hasUpdate == true) {
+        item.hasUpdate = Date.now() < next30DaysFromNow;
+      }
+    });
+  });
+});
```

Comment:
this logic should be handled in applicationCard itself

Reply:
This part of adding badges in the sidebar, how could it be handled from the application part?

Reply:
ah my bad it's almost the same as application can we make a util which handling this logic if it's duplicated?